src/go/rdctl/cmd/snapshotClean.go

```diff
@@ -0,0 +1,23 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/paths"
+	"github.com/spf13/cobra"
+)
+
+var snapshotCleanCmd = &cobra.Command{
+	Use:   "clean",
+	Short: "Clean snapshot state",
```

Comment:
Does this delete snapshots? Unwedge people when the snapshot lock file is stuck? An extended help might be useful.

Reply:
I added a `Long` field. [This comment](https://github.com/rancher-sandbox/rancher-desktop/pull/5496#discussion_r1339036356) is related.